packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -264,11 +263,11 @@ function Example() {
 ```
 
 Additionally, DialogTrigger accepts an `onOpenChange` prop which is triggered whenever the Dialog is opened or closed. 
-For more information on the `onOpenChange` prop see [DialogTrigger page](TODO LINK HERE).
+For more information on the `onOpenChange` prop see the [DialogTrigger docs](./DialogTrigger.html#events).
 
 ### Dismissable Dialogs
 
-Dismissable Dialogs support a `onDismiss` prop which is triggered whenever the Dialog's close button is clicked.
+Dismissable Dialogs support an `onDismiss` prop which is triggered whenever the Dialog's close button is clicked.
```

Comment:
You may notice that Dismissable Dialogs have their close button mispositioned in the examples now, looks like this is because the margin css for ActionButton is overriding the Dialog close button css in the docs (css import/order issue? doesn't happen in the storybook)

![image](https://user-images.githubusercontent.com/9661127/81735811-79318280-944a-11ea-87c7-55a766ebeabe.png)

I'll see if I can address this in a follow up pull